pagopa/io-app
# chore(IT Wallet): [SIW-727] Reset wallet migration

## Short description
This PR resets the wallet state with a migration.

## How to test
Activate the wallet and obtain a credential from `bundle/it-wallet` then switch to this branch. It should reset the state of the wallet.
